src/main/java/me/karakelley/http/filesystem/PublicDirectory.java

```diff
@@ -91,10 +87,22 @@ public Path getPath(String requestedResource) {
 
   public void updateFileContents(String path, byte[] contents) {
     File newFile = getPath(path).toFile();
-    try (FileOutputStream outputStream = new FileOutputStream(newFile, false)) {
+    try(FileOutputStream outputStream = new FileOutputStream(newFile, false)) {
       outputStream.write(contents);
     } catch (IOException e) {
       throw new RuntimeException(e);
     }
   }
+
+  public boolean isValidPath(String requestedPath) {
```

Comment:
Does any external client call this method? Does it need to be public? Or is it only public for purposes of testing?